ambari-server/src/main/java/org/apache/ambari/server/scheduler/ExecutionScheduleManager.java

```diff
@@ -566,6 +702,33 @@ public BatchRequestResponse getBatchRequestResponse(Long requestId, String clust
 
   }
 
+  protected RequestStatus abortRequestById(RequestExecution requestExecution, Long requestId) throws AmbariException {
+    LOG.debug("Aborting request " + requestId);
+    ResourceProvider provider =
+        ambariContext.getClusterController().ensureResourceProvider(Resource.Type.Request);
```

Comment:
AmbariContext is blueprint specific. Use AbstractControllerResourceProvider.getResourceProvider instead.